package.json

```diff
@@ -50,10 +50,8 @@
     "eslint-plugin-prettier": "3.1.0",
     "formik": "2.0.3",
     "humps": "2.0.1",
-    "immer": "3.2.0",
```

Comment:
Why delete immer? 🤔

Reply:
Because I didn't use it, because it feels weird to me that in the reducers I can mutate stuff, but meanwhile on my react components, I can't. I really like the idea of immer btw, but if I had to use, I prefer something that will stay the same across all the app

Reply:
You are not mutating stuff  🤔  you are changing an immutable copy of your state, it's a very big difference

Reply:
behind the scenes ofc. But in the code, in react components I have `list.map(item => ...`, and in the reducers I have to do this `list.forEach(item => item.foo = 'bar')`

Reply:
reducers and components are a whole diferent layer, so i'm still not seeing the point on removing immer just because "it looks like mutating stuff"

Reply:
Yeah, I'm trying to understand that, it just I don't want to add to many deps, but well, immer sounds like a really helpful one, I will integrate it again and refactor the 3 reducers I'm using. Maybe if I work a little bit more with it, I will get the gist of it and love it.

Reply:
And you took your time to include it and see if it was helpful, so I will try to be more open about that and take advantage of your work. You have been really helpful in all my other PR and this one.

Reply:
![image](https://user-images.githubusercontent.com/24686966/68963842-ac09ff00-07b6-11ea-97f2-8eef573694e5.png)

Reply:
I actually like it 👍